### PR TITLE
ES TN Fixes for #166

### PR DESCRIPTION
**Pull request by zoobereq**
# What does this PR do ?

Fixes the issue where the sentence-final period in sentences ending with domain is incorrectly normalized as part of the domain.  The PR also includes support for social media tags and updated tests.

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Have you signed your commits? Use ``git commit -s`` to sign.
- [x] Do all unittests finish successfully before sending PR?
   1) ``pytest`` or (if your machine does not have GPU) ``pytest --cpu`` from the root folder (given you marked your test cases accordingly `@pytest.mark.run_only_on('CPU')`).
   2) Sparrowhawk tests ``bash tools/text_processing_deployment/export_grammars.sh --MODE=test ...``
- [x] If you are adding a new feature: Have you added test cases for both `pytest` and Sparrowhawk [here](tests/nemo_text_processing).
- [x] Have you added ``__init__.py`` for every folder and subfolder, including `data` folder which has .TSV files?
- [x] Have you followed codeQL results and removed unused variables and imports (report is at the bottom of the PR in github review box) ?
- [x] Have you added the correct license header `Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.` to all newly added Python files?
- [x] If you copied [nemo_text_processing/text_normalization/en/graph_utils.py](nemo_text_processing/text_normalization/en/graph_utils.py) your header's second line should be `Copyright 2015 and onwards Google, Inc.`. See an example [here](https://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/text_normalization/en/graph_utils.py#L2).
- [x] Remove import guards (`try import: ... except: ...`) if not already done.
- [x] If you added a new language or a new feature please update the [NeMo documentation](https://github.com/NVIDIA/NeMo/blob/main/docs/source/nlp/text_normalization/wfst/wfst_text_normalization.rst) (lives in different repo).
- [x] Have you added your language support to [tools/text_processing_deployment/pynini_export.py](tools/text_processing_deployment/pynini_export.py).
  

**PR Type**:
- [ ] New Feature
- [x] Bugfix
- [ ] Documentation
- [x] Test

If you haven't finished some of the above items you can still open "Draft" PR.

### CODE REVIEW

File: tests/nemo_text_processing/es/data_text_normalization/test_cases_whitelist.txt [hunk 1]
```diff
@@ -2,6 +2,5 @@ el dr.~el doctor
 sr. rodriguez~señor rodriguez
 182 esq. toledo~ciento ochenta y dos esquina toledo
 da. ana pérez~doña ana pérez
-ee. uu.~estados unidos
```
Comment (tbartley94) [inline tests/nemo_text_processing/es/data_text_normalization/test_cases_whitelist.txt:5]: Why remove?

File: tests/nemo_text_processing/es/data_text_normalization/test_cases_telephone.txt [hunk 2]
```diff
@@ -7,7 +7,6 @@
 +1 (123) 123-5678~más uno uno dos tres uno dos tres cinco seis siete ocho
 1-800-go-u-haul~uno ochocientos go u haul
 llámame al 123-123-5678~llámame al uno dos tres uno dos tres cinco seis siete ocho
-123.123.0.40~doce tres punto uno dos tres punto cero punto cuatro cero~uno dos tres punto uno dos tres punto cero punto cuatro cero
```
Comment (tbartley94) [inline tests/nemo_text_processing/es/data_text_normalization/test_cases_telephone.txt:10]: Why remove?

File: nemo_text_processing/text_normalization/es/verbalizers/electronic.py [hunk 3]
```diff
@@ -58,21 +58,23 @@ def add_space_after_char():
...
+            pynutil.delete('protocol: "')
             + add_space_after_char() @ pynini.cdrewrite(graph_symbols, "", "", NEMO_SIGMA)
-            + pynutil.delete("\"")
+            + pynutil.delete('"')
         )
-        self.graph = (pynini.closure(protocol + pynini.accep(" "), 0, 1) + domain) | (
-            user_name + pynini.accep(" ") + pynutil.insert("arroba ") + domain
+
+        self.graph = (pynini.closure(protocol + NEMO_SPACE, 0, 1) + domain) | (
+            user_name + NEMO_SPACE + pynutil.insert("arroba ") + domain | (pynutil.insert("arroba ") + user_name)
```
Comment (tbartley94) [inline nemo_text_processing/text_normalization/es/verbalizers/electronic.py:76]: Just alias "arroba " with a variable name to avoid future spelling issues

**Review by tbartley94 — CHANGES_REQUESTED** [review]

File: tests/nemo_text_processing/es/data_text_normalization/test_cases_whitelist.txt (hunk 1 above)
Comment (zoobereq) [reply tests/nemo_text_processing/es/data_text_normalization/test_cases_whitelist.txt:5]: I think it was due to failing tests. I readded, retested, and all is fixed.

File: tests/nemo_text_processing/es/data_text_normalization/test_cases_telephone.txt (hunk 2 above)
Comment (zoobereq) [reply tests/nemo_text_processing/es/data_text_normalization/test_cases_telephone.txt:10]: Same as above. All fixed now.

File: nemo_text_processing/text_normalization/es/verbalizers/electronic.py (hunk 3 above)
Comment (zoobereq) [reply nemo_text_processing/text_normalization/es/verbalizers/electronic.py:76]: Done.

**Review by tbartley94 — APPROVED** [review]
LGTM

**Pull request closed without merging**